csharp/source/autowrap/csharp/common.d

```diff
@@ -0,0 +1,58 @@
+module autowrap.csharp.common;
+
+package template ArrayElementType(T : T[])
```

Comment:
What advantage does this have over `std.range.ElementType`?

Reply:
Dead code, removed.